src/kanata/mod.rs

```diff
@@ -242,7 +245,10 @@ static MAPPED_KEYS: Lazy<Mutex<cfg::MappedKeys>> =
 
 impl Kanata {
     /// Create a new configuration from a file.
-    pub fn new(args: &ValidatedArgs) -> Result<Self> {
+    pub fn new(
+        args: &ValidatedArgs,
+        #[cfg(feature = "passthru_ahk")] tx: Option<ASender<InputEvent>>,
```

Comment:
Please make (a) new `passthru_ahk` variant(s) instead of conditionally compiling different parameters in `new`.

Reply:
updated, but that didn't help with the mysterious error, so I've just added an extra None for the channel